app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -239,6 +244,29 @@ def update
       else
         return render status: :unprocessable_entity, json: { error: "Invalid parameter to be changed" }
       end
+    elsif group_type == UserGroup.group_types[:banned_competitors]
+      if params.key?(:endDate)
+        role.end_date = params.require(:endDate)
+      end
+
+      role.save!
```

Comment:
role.update can be used, but there are more parameters to be added in the next PR, so updating each inside if block and then role.save at end is ideal here.